app/(tabs)/notes.tsx

```diff
@@ -0,0 +1,86 @@
+import { Poppins_400Regular, useFonts } from '@expo-google-fonts/poppins';
+import { Redirect, useFocusEffect } from 'expo-router';
+import { useCallback, useState } from 'react';
+import { FlatList, SafeAreaView, StyleSheet } from 'react-native';
+import NoteItem from '../../components/NotesItem';
+import { colors } from '../../constants/colors';
+import type { Note } from '../../migrations/00003-createTableNotes';
+import type { NotesResponseBodyGet } from '../api/notes+api';
+import type { UserResponseBodyGet } from '../api/user+api';
+
+const styles = StyleSheet.create({
+  container: {
+    backgroundColor: colors.background,
+    height: '100%',
+  },
+  text: {
+    color: colors.text,
+  },
+  list: {
+    marginTop: 30,
+  },
+});
+
+export default function App() {
```

Comment:
The function name should match the filename (but in PascalCase)

(for all function components in the PR)

Reply:
[5a205ab](https://github.com/upleveled/expo-example-fall-2024-atvie/pull/1/commits/5a205abbb49c3eb76982f4177b159bfbca6edb9f)